lib/chef_zero/endpoints/acls_endpoint.rb

```diff
@@ -22,6 +22,17 @@ def get(request)
         end
         acls = FFI_Yajl::Parser.parse(get_data(request, acl_path), :create_additions => false)
         acls = ChefData::DataNormalizer.normalize_acls(acls)
+        if request.query_params["detail"] == "granular"
+          acls.each do |perm, ace|
+            acls[perm]["actors"] = []
```

Comment:
This is mirroring the behavior that we added to the Chef Server, where if you request granular `actors` will be empty but `users` and `clients` will not. 

This prevents someone from using a modified version of the GET response to PUT an acl update, and getting an error because Chef Server won't except a PUT body with both actors and users/clients. 

It's a bit awkward, but is the path least likely to break existing tooling outside of what we maintain - the most common method of modifying ACLs is to request the ACLs, modify a single ACE, and PUT it back to the server.